inc/itilsolution.class.php

```diff
@@ -605,4 +605,44 @@ public function showSummary(CommonITILObject $item) {
          echo Html::scriptBlock($js);
       }
    }
+
+   /**
+    * {@inheritDoc}
+    * @see CommonDBTM::getSpecificValueToDisplay()
+    */
+   static function getSpecificValueToDisplay($field, $values, array $options = []) {
+
+      if (!is_array($values)) {
+         $values = [$field => $values];
+      }
+
+      switch ($field) {
+         case 'status':
+            return CommonITILValidation::getStatus($values[$field]);
+            break;
+      }
+
+      return parent::getSpecificValueToDisplay($field, $values, $options);
+   }
+
+   /**
+    * {@inheritDoc}
+    * @see CommonDBTM::getSpecificValueToSelect()
+    */
+   static function getSpecificValueToSelect($field, $name = '', $values = '', array $options = []) {
+
+      if (!is_array($values)) {
+         $values = [$field => $values];
+      }
+
+      switch ($field) {
+         case 'status':
+            $options['display'] = false;
+            $options['value'] = $values[$field];
+            return CommonITILValidation::dropdownStatus($name, $options);
```

Comment:
You cannot rely on that here; labels willnot be correct:
- "Waiting for approval" is not correct,
- "Granted" is wrong.

Reply:
I agree for "Granted" (I did not saw it as is translated ac "Acceptée" in french), but why "Waiting for approval" is not correct ?

Reply:
The "approval" term does not seems to me the best choice here. I do not have anything else to porpose, so if "Granted" has been fixed it's ok for me.

Reply:
It has been fixed.